capstone/src/main/java/com/step902020/capstone/IndividualController.java

```diff
@@ -20,14 +20,28 @@
   @Autowired
   private IndividualRepository individualRepository;
 
-  /** Find an individual's profile information by email */
+  @Autowired
+  private EventRepository eventRepository;
+
+  /** Find an individual's profile information by email
+   * @param email user's email from IAP header
+   * @return list of individuals with the same email
+   */
   @GetMapping("get-individual")
-  public List<Individual> getIndividual(@RequestHeader("X-Goog-Authenticated-User-Email") String email) {
-    return this.individualRepository.findByEmail(email.substring(20));
+  public List<Individual> getIndividual(@RequestHeader("X-Goog-Authenticated-User-Email") Optional<String> email) {
+    return this.individualRepository.findByEmail("jennysheng@google.com");
```

Comment:
Should this be `email.orElse("jennysheng@google.com")`?

Reply:
Yes added this for testing locally but will fix in the next commit when I fix merge conflicts